app/views/soft-close/no-confirm.html

```diff
@@ -0,0 +1,23 @@
+{% extends "_wizard-page.html" %}
+{% block pageNavigation %}{% endblock %}
+{% set title = 'You have chosen not to receive an alert' %}
```

Comment:
```suggestion
{% set title = 'You will not get an alert' %}
```
I wonder if this page should not have the green success panel

Something pared back like this?
![Screenshot 2021-12-13 at 10 14 14](https://user-images.githubusercontent.com/319055/149361867-3eb906d0-2cdc-4ebc-985a-92ab7b6d8083.png)

Reply:
Yep, good point!

Reply:
![Screenshot 2022-01-13 at 16 02 04](https://user-images.githubusercontent.com/66423623/149364958-fa02c7e6-b604-4dfb-8679-4d19e9bf502c.png)
 I'll leave it like this for now, as pairing with James tomorrow